derex/runner/mysql.py

```diff
@@ -80,52 +106,68 @@ def show_databases() -> List[Tuple[str, int, int]]:
     return databases_tuples
 
 
-def show_users() -> Optional[Tuple[Tuple[str, str, str]]]:
-    """List all mysql users.
-    """
-    client = get_mysql_client()
+def list_users() -> Optional[Tuple[Tuple[str, str, str]]]:
+    """List all mysql users."""
+    client = get_system_mysql_client()
     client.execute("SELECT user, host, password FROM mysql.user;")
     users = cast(Tuple[Tuple[str, str, str]], client.fetchall())
     return users
 
 
 def create_database(database_name: str):
-    """Create a database if doesn't exists"""
-    client = get_mysql_client()
+    """Create a database if doesn't exists."""
+    client = get_system_mysql_client()
     logger.info(f'Creating database "{database_name}"...')
     client.execute(f"CREATE DATABASE `{database_name}` CHARACTER SET utf8")
     logger.info(f'Successfully created database "{database_name}"')
 
 
 def create_user(user: str, password: str, host: str):
-    """Create a user if doesn't exists"""
-    client = get_mysql_client()
+    """Create a user if doesn't exists."""
+    client = get_system_mysql_client()
     logger.info(f"Creating user '{user}'@'{host}'...")
     client.execute(f"CREATE USER '{user}'@'{host}' IDENTIFIED BY '{password}';")
     logger.info(f"Successfully created user '{user}'@'{host}'")
 
 
 def drop_database(database_name: str):
-    """Drops the selected database"""
-    client = get_mysql_client()
+    """Drops the selected database."""
+    client = get_system_mysql_client()
     logger.info(f'Dropping database "{database_name}"...')
     client.execute(f"DROP DATABASE IF EXISTS `{database_name}`;")
     logger.info(f'Successfully dropped database "{database_name}"')
 
 
 def drop_user(user: str, host: str):
-    """Drops the selected user"""
-    client = get_mysql_client()
+    """Drops the selected user."""
+    client = get_system_mysql_client()
     logger.info(f"Dropping user '{user}'@'{host}'...")
     client.execute(f"DROP USER '{user}'@'{host}';")
     logger.info(f"Successfully dropped user '{user}'@'{host}'")
 
 
+def execute_root_shell(command: Optional[str]):
+    """Open a root shell on the mysql database. If a command is given
+    it is executed."""
+    args = [
+        "run",
```

Comment:
This will work only if the mysql container is already running. We can speed things up by not creating a new container, using `exec` instead of `run`